packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -4130,6 +4150,7 @@ namespace Ifpack2 {
       //sqrt the norms for the caller's use.
       if (is_norm_manager_active) norm_manager.finalize();
 
+      IFPACK2_BLOCKTRIDICONTAINER_TIMER_FENCE(typename ImplType<MatrixType>::execution_space)
```

Comment:
This is just after the end of a serial parallel for, this should not be required?